### PR TITLE
Filter cta data by collateral price

### PR DESCRIPTION
**Pull request by Chepelau**

### CODE REVIEW

File: src/cta/cta.py [hunk 1]
```diff
@@ -53,11 +53,20 @@ def get_cta_message(data: pd.DataFrame) -> str:
     data = data[
         data['debt_to_supply_ratio'] >= 0.75
     ] 
+
+    if collateral_token_price is not None:
```
Comment (lukaspetrasek) [inline src/cta/cta.py:57]: Add a warning log here in case `collateral_token_price` is None pls 🙏🏼

File: src/cta/cta.py [hunk 2]
```diff
@@ -188,6 +197,8 @@ def generate_and_store_ctas(session: Session):
     for ix, selection in enumerate(tokens_combos):
         logging.info(f'Generating cta for: {selection}')
 
+        collateral_price = tokens_prices.get(selection.collateral.address)
+
         # Df containing liquidity and liquidable debt 
         df = get_cta_data(
```
Comment (lukaspetrasek) [inline src/cta/cta.py:203]: Pls name it descriptively.

**Review by lukaspetrasek — CHANGES_REQUESTED** [review]

File: src/cta/cta.py [hunk 3]
```diff
@@ -34,7 +34,7 @@ def generate_message(
     """
 
 
-def get_cta_message(data: pd.DataFrame) -> str:
+def get_cta_message(data: pd.DataFrame, collateral_token_price: float | None) -> str:
```
[the remark this answers is not in the document]
Comment (Chepelau) [reply src/cta/cta.py:37]: `import pandas as pd` is the suggested way to import `pandas` library, as per `pandas` documentation: https://pandas.pydata.org/docs/user_guide/10min.html

File: src/cta/cta.py (hunk 1 above)
Comment (Chepelau) [reply src/cta/cta.py:57]: Added it in the `generate_and_store_ctas` function as logging it here would not provide any information about the address of the token for which we're missing a price. That would require additional argument for the `get_cta_message` and that seems kinda redundant.

File: src/cta/cta.py (hunk 2 above)
Comment (Chepelau) [reply src/cta/cta.py:203]: Renamed `df` to `cta_data`

**Review by lukaspetrasek — APPROVED** [review]

**Pull request merged**